### PR TITLE
#160847968 Refactor deployment scripts

### PR DESCRIPTION
**Pull request by HarithJ**
#### What does this PR do?
- rok8s-scripts config file needs to be refactored such that all the services are listed inside one key, instead of creating multiple SERVICES keys
- Delete nginx-ingress-controller as it already gets created when mandatory command is run
- create single tls secret object

#### Description of Task to be completed?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#160847968

### CODE REVIEW

File: deploy/nginx.service.yml.tpl [hunk 1]
```diff
@@ -1,23 +1,22 @@
-# This is the backend service
-apiVersion: v1
 kind: Service
+apiVersion: v1
```
Comment (Thegaijin) [inline deploy/nginx.service.yml.tpl]: Is there a reason for switching the kind and apiVersion positions?

Comment (HarithJ) [reply deploy/nginx.service.yml.tpl]: no, it does not matter. I have copied this file from kubernetes.

File: deploy/nginx.service.yml.tpl [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-    name: http
-    targetPort: 80
-  - port: 443
-    name: https
-    targetPort: 443
   selector:
-    # Selects nginx-ingress-controller pods
     app.kubernetes.io/name: ingress-nginx
     app.kubernetes.io/part-of: ingress-nginx
+  ports:
```
Comment (Thegaijin) [inline deploy/nginx.service.yml.tpl]: Similar question to the one above regarding the ports, is there a reason for moving the configurations down?

Comment (HarithJ) [reply deploy/nginx.service.yml.tpl]: no, it does not matter. I have copied this file from kubernetes.

**Pull request merged**